xmlDrawing.go

```diff
@@ -325,6 +325,28 @@ type xlsxChart struct {
 	R   string `xml:"xmlns:r,attr"`
 }
 
+// decodeChoice defines the structure used to deserialize the mc:Choice element.
```

Comment:
This code block should be move into `xmlDecodeDrawing.go`.

Reply:
I moved this code block, you can take a look again.